extensions/sql-database-projects/src/dialogs/updateProjectFromDatabaseDialog.ts

```diff
@@ -572,7 +572,7 @@ export class UpdateProjectFromDatabaseDialog {
 			projectFilePath: this.projectFileDropdown!.value! as string,
 			extractTarget: mapExtractTargetEnum(<string>this.folderStructureDropDown!.value),
 			targetScripts: [],
-			dataSchemaProvider: '',
+			dataSchemaProvider: this.project!.getProjectTargetVersion(),
```

Comment:
does the schema compare extension already do this or does a similar fix need to happen there as well?

Reply:
not sure if the `dataSchemaProvider` gets set later, but this place in schema compare might also need to be updated: https://github.com/microsoft/azuredatastudio/blob/d69e5b97df19ec32dc41fd186c83b819e2ed72af/extensions/schema-compare/src/schemaCompareMainWindow.ts#L150-L163

Reply:
So, in the above spot, we don't have any context about dataSchemaProvider. We just have the projectFilePath.
The data schema provider gets appropriately set when SchemaCompareDialog opens and "Ok" button is clicked. So, this scenario (schema compare started from project context) works as expected.